server/lib/nl/fulfillment/fulfiller.py

```diff
@@ -43,6 +43,13 @@ def fulfill(uttr: Utterance, explore_mode: bool = False) -> PopulateState:
   # Construct a common PopulateState
   state = PopulateState(uttr=uttr)
 
+  da_cls = futils.classifications_of_type(
+      uttr.classifications, dtypes.ClassificationType.DETAILED_ACTION)
+  if da_cls:
+    uttr.counters.info('fulfill_detailed_action_querytypes',
+                       utils.get_action_verbs(uttr))
+    return state
```

Comment:
nit:  how about making it a bit concise like

```
detailed_action = utils.get_action_verbs(uttr)
if detailed_action:
  ...
  return state
```

Reply:
Done